Hunting Queries/AzureDevOps/AAD Conditional Access Disabled.yaml

```diff
@@ -0,0 +1,19 @@
+id: c78a3845-37d9-448e-a8cd-e9543f00bcc5
+name: Azure DevOps- AAD Conditional Access Disabled 
+description: |
+  'This hunting query identifies Azure DevOps activities where organization AADConditionalAccess policy disable by the admin'
```

Comment:
We should probably add the data connectors details – somewhat like below. Please look at other hunting queries for reference.
requiredDataConnectors:
  - connectorId: AzureMonitor(AzureDevOpsAuditing)
    dataTypes:
      - AzureDevOpsAuditing

Reply:
Thanks @aprakash13 I pushed an updated to accommodate this